torch/csrc/jit/codegen/cuda/ir_nodes.cpp

```diff
@@ -851,6 +871,23 @@ bool IterDomain::maybePartial() const {
   return !start()->isZeroInt() || !stop()->sameAs(extent());
 }
 
+Val* IterDomain::stopOffset() const {
+  // Returns sub(extent(), stop()) but try to simplify it if possible
+
+  if (extent()->sameAs(stop())) {
+    return new Int(0);
+  }
+
+  // If stop() == extent() - a, which is common, just returns a
```

Comment:
I think just having the stop offset makes sense for now since the only use case of `stop` is to express how much we cut from the end of a given domain. I'll try to refactor it.

Reply:
Updated